cmd/dep/init_test.go

```diff
@@ -144,3 +146,115 @@ const Qux = "yo yo!"
 	// vendor should have been created & populated
 	h.MustExist(h.Path("src/" + root + "/vendor/github.com/Sirupsen/logrus"))
 }
+
+// TestCaseInsentitive is test for Windows. User can case-insenstive GOPATH.
+// This should work even though set GOPATH=C:/path/To/GoPath.
```

Comment:
Does this test rely on GOPATH being set to a certain value?
I don't see how this actually tests the change you have made here.